### PR TITLE
Make updates as per wikipedia suggestions

### PR DESCRIPTION
**Pull request by Adina-Rahim**
I have revised the suggested entries to ensure their compatibility with Wikipedia.

### CODE REVIEW

File: people.json [hunk 1]
```diff
@@ -4,21 +4,19 @@
...
             "Machine Learning",
-            "Deep Learning for Computer Vision",
             "Data Science",
             "Data Mining",
-            "Association Rule Mining",
+            "Association Rule Learning",
             "Sequential Pattern Mining",
             "Link Analysis",
-            "Analysis of Large Graphs: Trust Rank and Webspam",
+            "TrustRank and WebSpam",
```
[the remark this answers is not in the document]
Comment (rowleya) [reply people.json:16]: Sorry, my suggestion wasn't that clear - I meant them to be separate (as TrustRank is a Wikipedia thing)

**Comment by ianhinder** [comment]
@rowleya -- thanks for reviewing this.  One note: as the person responsible for the deployment, I'd prefer if merges to master were left to me, since I am currently the only one who gets a notification if the deployment fails, and the only one who can log in, poke around, and fix it.  In future, I'd like to "democratise" this much more, but until we move to a more sane deployment mechanism, it's probably best if I handle the merges to master!

**Comment by rowleya** [comment]
Ah OK - didn't realise you were also deploying things so yes I can leave it to you to do the merges :).

**Comment by ianhinder** [comment]
It's automatically deployed to the live production site on merges to master.  We should probably restrict this by technical means (branch protection rules) until the deployment code is rewritten, though I don't want to come across as possessive, and everyone is an admin anyway, so it might not even work!

**Comment by rowleya** [comment]
If testing is enough to make sure it all works i.e. successful test == successful deployment, and branch protection ensure successful test before merging, that probably makes it good enough and means anyone merging to master is OK (assuming deployment isn't something that should be limited e.g. because of costs or something).  People should not be overriding the protection rules once in place, or at least when they do they are basically saying they are responsible for the deployment and any errors in it at that point!

Of course this is currently in flux so until that is the case you are right to ask that only yourself merges to master and triggers deployment at this point!

**Pull request merged**